synapse/handlers/auth.py

```diff
@@ -161,6 +161,10 @@ def __init__(self, hs):
         self._sso_auth_confirm_template = load_jinja2_templates(
             hs.config.sso_redirect_confirm_template_dir, ["sso_auth_confirm.html"],
         )[0]
+        self._sso_account_deactivated_template = load_jinja2_templates(
```

Comment:
Why not load the file with `self.read_file` since there's no parameter to use when rendering the template? Like this: https://github.com/matrix-org/synapse/blob/0de9f9486a242c8dbee4b9bc65cad166b863094f/synapse/config/saml2_config.py#L166-L172

Reply:
Because I did not know that existed! I'll update it.